documentation/airport-format.md

```diff
@@ -1,309 +1,223 @@
----
-title: Airport format
----
-[back to index](index.html)
+# Airport Format
 
-# Airport format
+* [NOTICE](## NOTICE)
+* [Template](## Template)
+* [Comments](## Comments)
+    * [Property Descriptions](### Property Descriptions)
+        * [Latitude, Longitude, Elevation](#### Latitude, Longitude, Elevation)
+        * [ICAO and IATA identifiers](#### ICAO and IATA identifiers)
+
+## NOTICE
 
 The airport JSON file must be in `assets/airports`; the filename
 should be `icao.json` where `icao` is the lowercase four-letter ICAO
 airport code, such as `ksfo` or `kmsp`.  If this is a new airport, there
-should also be an entry added to the end of [`airportLoadList.js`](../src.assets/scripts/airport/airportLoadList.js)
+should also be an entry added to [`airportLoadList.js`](../src.assets/scripts/airport/airportLoadList.js) in alphabetical order.
 See the comments for information on the correct structure to use.
+
+## Template
```

Comment:
change this heading to `Example` and add a note between the heading and the code block that this is an abbreviated example of the `klas.json` file.  you could also add a link to that file if you like.